released/packages/coq-parseque/coq-parseque.0.1.2/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: ["G. Allais" "W. Meier"]
+dev-repo: "git+https://github.com/gallais/parseque"
+bug-reports: "https://github.com/gallais/parseque/issues"
+homepage: "https://github.com/gallais/parseque"
+authors: ["G. Allais"]
+license: "GPL-3.0-or-later"
+build: [
+  [make "compile"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml"
```

Comment:
Is there some actual OCaml code to compile? Or is it a spurious dependency?

Also, the package is missing a lot of metadata. At the very least, there should be
```
tags: [
  "date:2023-09-27"
  "logpath:parseque"
]
```

But there could be more: `category`, `keyword`.